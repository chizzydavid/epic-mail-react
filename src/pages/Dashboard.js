import React, { Component } from 'react'
import Navbar from '../components/shared/Navbar';
import Footer from '../components/shared/Footer';
import MessagesContainer from '../components/MessagesContainer';

class Dashboard extends Component {

  render() {
    return (
      <React.Fragment>
        <Navbar />
        <MessagesContainer />
        <Footer />
      </React.Fragment>
    )
  }
}

export default Dashboard;
