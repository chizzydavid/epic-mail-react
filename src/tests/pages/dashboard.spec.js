import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '../../pages/Dashboard';

describe('Component: Dashboard', () => {
  it('should render correctly', () => {

    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});