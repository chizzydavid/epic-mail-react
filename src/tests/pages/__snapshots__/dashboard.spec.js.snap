// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Dashboard should render correctly 1`] = `ShallowWrapper {}`;
